eth/contracts/TreasureHuntCreator.sol

```diff
@@ -4,8 +4,8 @@ import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
 contract TreasureHuntCreator is Ownable {
   event ChapterCompleted(uint indexed completedChapter, address indexed player);
 
-  mapping (uint256 => address[]) public _chapterToPlayers;
-  mapping (address => uint256) public _playerToCurrentChapter;
+  mapping (uint96 => address[]) public _chapterToPlayers;
```

Comment:
why `uint96`?

Reply:
In this way we can have up to `79228162514264337593543950336` chapters.

Nah it's just OCD, since we return a `uint256[]` for the leaderboard, and an address is 160 bits, we have 96 bits left that we can use for the chapters. I can put a `uint8` if you want (and storing data would be cheaper).